apisix/http/router/radixtree_host_uri.lua

```diff
@@ -94,7 +94,11 @@ local function create_radixtree_router(routes)
     host_router = nil
 
     for _, route in ipairs(routes or {}) do
-        push_host_router(route, host_routes, only_uri_routes)
+        local status = core.table.try_read_attr(route, "value", "status")
+        -- check the status
+        if status == 1 then
```

Comment:
The existing old data has no field "status". 

we need to make sure it works fine for this case.

`if not status or status == 1 then`, this way is safer.

Reply:
I took a look at the apisix sync data from etcd,
https://github.com/apache/apisix/blob/c41aa8534a18f43ef0c3cd6780f106595f9f9b88/apisix/core/schema.lua#L53-L61

after `validator(json) `, the existing old data will has default `status` value. 

but i agree your point, add `if not status` is more safer:-)